### PR TITLE
Update SDK and libraries

### PR DESCRIPTION
**Pull request by paolorotolo**
Please test and merge.

### CODE REVIEW

**Comment by emartynov** [comment]
I bet Travis will fail :)

File: app/build.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ android {
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 26
+        targetSdkVersion 27
```
Comment (emartynov) [inline app/build.gradle:35]: Don't forget to update Travis yaml

File: build.gradle [hunk 2]
```diff
@@ -43,10 +43,10 @@ allprojects {
         google()
     }
 
-    ext.buildToolsVer = '27.0.1'
+    ext.buildToolsVer = '27.0.2'
```
Comment (emartynov) [inline build.gradle:46]: Travis!

File: build.gradle [hunk 3]
```diff
@@ -43,10 +43,10 @@ allprojects {
...
 
-    ext.androidSupportVer = '26.1.0'
-    ext.googlePlayServicesVer = '11.6.2'
+    ext.androidSupportVer = '27.0.2'
```
Comment (emartynov) [inline build.gradle:48]: As soon as we start with Kotlin it will be huge headache :)

**Review by emartynov — DISMISSED** [review]
Check the Travis and please ignore comment about support library

**Comment by paolorotolo** [comment]
Nice one @emartynov 👍

**Pull request merged**